cpu/nrf52/include/periph_cpu.h

```diff
@@ -40,6 +40,30 @@ extern "C" {
 #define SPI_MISOSEL         (dev(bus)->PSEL.MISO)
 /** @} */
 
+/**
+ * @brief   Override I2C speed settings
+ * @{
+ */
+#define HAVE_I2C_SPEED_T
+typedef enum {
+    I2C_SPEED_LOW       = 0x01,                         /**< not supported */
```

Comment:
I think it would be better to define them as 0xFF, 0xFE, ... if those values are not supported rather than 0x01, 0x02, ...